Description: typo-in-manual-page usr/share/man/man1/iio_attr.1.gz availble available
Author: "A. Maitland Bottoms" <bottoms@debian.org>

--- a/man/iio_attr.1.in
+++ b/man/iio_attr.1.in
@@ -84,7 +84,7 @@
 Read and Write IIO Debug attributes
 .TP
 .B \-S, \-\-Scan
-Scan for available IIO contexts, optional arg of specific backend(s) 'ip', 'usb' or 'ip:usb'. If no argument is given, it checks all that are availble.
+Scan for available IIO contexts, optional arg of specific backend(s) 'ip', 'usb' or 'ip:usb'. If no argument is given, it checks all that are available.
 .TP
 .B \-h, \-\-help
 Tells
